cns/restserver/restserver.go

```diff
@@ -933,6 +934,24 @@ func (service *HTTPRestService) createOrUpdateNetworkContainer(w http.ResponseWr
 					break
 				}
 			}
+		} else if req.NetworkContainerType == cns.AzureContainerInstance {
+			// try to get the saved nc state if it exists
+			service.lock.Lock()
+			existing, ok := service.state.ContainerStatus[req.NetworkContainerid]
+			service.lock.Unlock()
+
+			// create/update nc only if it doesn't exist or it exists and the requested version is different from the saved version
+			if ok && existing.VMVersion != req.Version {
```

Comment:
does it matter if we check for != versus < version?

Reply:
I am assuming != will take of that.